note/doc/index.md

```diff
@@ -0,0 +1,73 @@
+```js script
+import { html } from 'lit';
+import '@divriots/dockit-core/note/dockit-note.define.js';
+```
+
+# dockit-note
+
+Component used to call out information in documentation pages.
+
+## Setup
+
+```js
+import '@divriots/dockit-core/note/dockit-note.define.js';
+```
+
+## Demo
+
+```html preview-story
+<dockit-note>
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit
+</dockit-note>
+```
+
+### Variants
+
+#### Info
+
+```html preview-story
+<dockit-note variant="info">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit
+</dockit-note>
+```
+
+#### Warning
+
+```html preview-story
+<dockit-note variant="warning">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit
+</dockit-note>
+```
+
+#### Error
+
+```html preview-story
+<dockit-note variant="error">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit
+</dockit-note>
+```
+
+### Note with custom color
+
+```html preview-story
+<dockit-note color="BurlyWood">
```

Comment:
I think we should follow always the same method for passing CSS props via custom CSS prop instead of exposing a prop/attribute on the element